packages/cpp/ArmoniK.Api.Client/header/sessions/SessionsClient.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include "sessions_common.pb.h"
+#include "sessions_service.grpc.pb.h"
+
+namespace armonik {
+namespace api {
+namespace client {
+
+/**
+ * Session client wrapper
+ */
+class SessionsClient {
+public:
+  explicit SessionsClient(std::unique_ptr<armonik::api::grpc::v1::sessions::Sessions::StubInterface> stub)
+      : stub(std::move(stub)){};
+
+  /**
+   * Create a new session
+   * @param default_task_options Default task options for the session
+   * @param partitions Partitions the session will be able to send tasks to
+   * @return Session id
+   */
+  std::string create_session(const armonik::api::grpc::v1::TaskOptions &default_task_options,
```

Comment:
I think you can take the `default_task_options` by value:
```suggestion
  std::string create_session(armonik::api::grpc::v1::TaskOptions default_task_options,
```

Reply:
Should I do the same for the partitions and for the list_sessions parameters as well do you reckon ?